### ISSUE TITLE
Add OSMAnd File Format

### ISSUE THREAD

**Issue by wonderchook** [hotosm/hot-exports#77]
Currently we rely on other services to update OSMAnd during activations. Allowing users to update themselves through the export would help make activations more effective.

https://code.google.com/p/osmand/

**Comment by bgirardot** [comment]
@mataharimhairi You are correct. I searched issues for .obf and didn't see any, I should have searched for OsmAnd instead.
I'll close my other issues.
@dodobas I think this one needs to be high priority in light of Missing Maps popularity and their use of OsmAnd in their workflow and our regular activations coming to rely on it more and more.
The ease and benefit of using the Export server to generate these files would really make a difference.

**Comment by xivk** [comment]
The advantages of having this in the export(s):
- Not everybody in the field-teams has the skills or knowledge to create these files. 
- They would be available for download on Android without the use of another machine (!). There are people who don't have another machine, just an Android device.
- No need to distribute the files among team members.
- No need to connect your device to your other machine or to manually put it somewhere for download.

Just to say this saves a lot of time and even though it's almost trivial to create these files, having it in the export has it's advantages.

**Comment by harry-wood** [comment]
A disadvantage is that we'll be using more disk space for new file types which may not have anything to do with what the user was wanting (we need tickboxes! #78 )
